nanoFramework.Device.Bluetooth/BluetoothLEDevice.cs

```diff
@@ -528,6 +591,18 @@ internal void OnEvent(BluetoothEventCentral btEvent)
             }
         }
 
+        internal void OnEvent(BluetoothEventSesssion btEvent)
+        {
+            //Debug.WriteLine($"# BluetoothLEdevice OnEvent session, {btEvent.type} status:{btEvent.status}");
+
+            switch (btEvent.type)
```

Comment:
What is the interest of the switch if there is only the default? :-) Do you plan to add more later on?

Reply:
I thought there would be when it was done. I'll remove switch.